.github/workflows/sync-integration-templates.yaml

```diff
@@ -30,10 +30,11 @@ jobs:
                       popd
 
                       cp -r /tmp/nango-temp/nango-integrations/dist ./$integration/dist
+                      cp -r /tmp/nango-temp/nango-integrations/.nango ./$integration/.nango
 
                       rm -rf /tmp/nango-temp
 
-                      aws s3 sync $dir s3://${{ secrets.AWS_BUCKET_NAME }}/$integration/
+                      aws s3 sync $integration s3://${{ secrets.AWS_BUCKET_NAME }}/$integration/
```

Comment:
does the upload was broken before? it feels like an incompatible change no?

Reply:
Yeah. Was broken before.